exercises/roman-numerals/runtests.jl

```diff
@@ -0,0 +1,36 @@
+using Base.Test
+
+include("roman-numerals.jl")
+
+# NOTE: Got that one from Python's testset
+samples = Dict(
+    1 => "I",
+    2 => "II",
+    3 => "III",
+    4 => "IV",
+    5 => "V",
+    6 => "VI",
+    9 => "IX",
+    27 => "XXVII",
+    48 => "XLVIII",
+    59 => "LIX",
+    93 => "XCIII",
+    141 => "CXLI",
+    163 => "CLXIII",
+    402 => "CDII",
+    575 => "DLXXV",
+    911 => "CMXI",
+    1024 => "MXXIV",
+    3000 => "MMM"
+)
+
+@testset "basic transformations" begin
+    for sample in samples
+        @test to_roman(sample[1]) == sample[2]
+    end
+end
+
+@testset "error handling" begin
+    @test_throws ErrorException to_roman(0)
+    @test_throws ErrorException to_roman(3001)
```

Comment:
A solution that handles numbers higher than 3000 would still be valid, even if it's not required and we don't test for it. Imo failing the tests if a solution goes beyond 3000 isn't ideal.

I agree that there should be some sort of error for numbers that it can't handle, though. Perhaps having a hint or a constant for the maximum number would be a bit less restrictive?